### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by strtob** [asmaier/ImmoSpider#24]
Hi,

This is really a nice script! It seems due to newest restriction of IS this do not work anymore. Is this right? I've startet the script but it don't find any items. 

Thanks!

**Comment by psevdoname** [comment]
Yeah, it doesn't work. I wonder if the approach from this project can be implemented https://github.com/orangecoding/fredy

**Comment by orangecoding** [comment]
@psevdoname Maintainer of Fredy here. What'S the reason you just don't use Fredy? ;)
